### DISCUSSION TITLE
GeneralModelChangeEventType subscription not working

### DISCUSSION THREAD

**Discussion by rabi-kumar** [eclipse-milo/milo#1027] [General]
@kevinherron Thank you for the awesome project. Currently, I am producing GeneralModelChangeEventType as per [Issue676](https://github.com/eclipse/milo/issues/676) in the node which has been very helpful. 

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/42206304/184362486-4bb753cc-e26f-4b5d-8aca-40b258dbbba6.png">

The event can be seen in UAExpert as well

<img width="1690" alt="image" src="https://user-images.githubusercontent.com/42206304/184363343-6aa99f00-c6c1-40b9-a227-39d4501893a4.png">

Now I am trying to understand how would the event subscription for GeneralModelChangeEventType would work. For this, I am modifying [EventSubscriptionExample.java](https://github.com/eclipse/milo/blob/master/milo-examples/client-examples/src/main/java/org/eclipse/milo/examples/client/EventSubscriptionExample.java).. I am subscribing to NodeId (1, 55443) and only readvalueid has been changed, but I am getting the following error:

<img width="1431" alt="image" src="https://user-images.githubusercontent.com/42206304/184364943-8a49da51-f865-4e9e-8c98-7ceee6a6bcd8.png">

Upon debugging, I see that the execution stops at setEventConsumer and then returns null

<img width="924" alt="image" src="https://user-images.githubusercontent.com/42206304/184367865-99045db5-4381-4f33-ad89-a9f9eab6dbb9.png">

Could you please help me with this?

**Comment by kevinherron** [comment]
You shouldn't change anything in `EventSubscriptionExample`, the Server node is still the Event Notifier, and the example is already set up to subscribe to any kind of event.

**Comment by rabi-kumar** [comment]
Hi @kevinherron Thank you for the suggestion. Now, what I am trying to do is I want to create another node that will act as an Event Notifier for the GeneralModelChangeEventType inside it, and change the `EventSubscriptionExample` which will subscribe to this single node. Do you think this is possible?

Ultimately the idea is I would browse the whole address space and would identify the event notifier node with GeneralModelChangeEvent inside it and would subscribe to these.
